library/src/main/java/com/google/android/cameraview/CameraView.java

```diff
@@ -407,13 +411,16 @@ public void takePicture() {
         mImpl.takePicture();
     }
 
-    private class CallbackBridge implements CameraViewImpl.Callback {
+    private static class CallbackBridge implements CameraViewImpl.Callback {
```

Comment:
Important for memory leak: non-static inner classes implicitly hold references to their parent.  This was causing a big leak.  Making it static fixes that.